modules/monitoring/templates/smokey-loop.conf

```diff
@@ -6,4 +6,4 @@ description "Smokey loop outputs JSON which is consumed by monitoring"
 start on runlevel [2345] and stopped $UPSTART_JOB
 respawn
 
-exec /opt/smokey/tests_json_output.sh /tmp/smokey.json
```

Comment:
Have tested this in integration, the @environment comes out as "production", which is probably right, but not helpful, as I want to have the GOV.UK environment here. It also doesn't have a ending newline, but that can be fixed by removing the - before the closing % sign.

Does anyone know what I can put here to get the GOV.UK environment as a string?